packages/footer/CHANGELOG.md

```diff
@@ -36,6 +37,10 @@
 
 ## Release History
 
+### v2.1.7
+
+- Added an aria-label attribute to the nav element
+
```

Comment:
This needs one extra line break before the version underneath, `npm run test` will probably fail due to spacing right now.